activitysim/abm/models/non_mandatory_tour_frequency.py

```diff
@@ -418,13 +428,14 @@ def non_mandatory_tour_frequency(
     if estimator:
         # make sure they created the right tours
         survey_tours = estimation.manager.get_survey_table("tours").sort_index()
-        non_mandatory_survey_tours = survey_tours[
-            survey_tours.tour_category == "non_mandatory"
-        ]
-        assert len(non_mandatory_survey_tours) == len(non_mandatory_tours)
-        assert non_mandatory_survey_tours.index.equals(
-            non_mandatory_tours.sort_index().index
-        )
+        # FIXME below check needs to remove the pure-escort tours from the survey tours table
```

Comment:
Would it be better to optionally skip this section of code instead of simply commenting them out? What is the use case for commenting them out?

Reply:
Agreed.  Removed outdated comment.